packages/office-ui-fabric-react/src/components/GroupedList/GroupSpacer.tsx

```diff
@@ -1,23 +1,11 @@
 import * as React from 'react';
-import { styled, classNamesFunction } from '../../Utilities';
-import { IGroupSpacerProps, IGroupSpacerStyleProps, IGroupSpacerStyles } from './GroupSpacer.types';
-import { getStyles } from './GroupSpacer.styles';
-
-const getClassNames = classNamesFunction<IGroupSpacerStyleProps, IGroupSpacerStyles>();
+import { IGroupSpacerProps } from './GroupSpacer.types';
 
 export const SPACER_WIDTH = 32;
 
-export const GroupSpacer = styled<IGroupSpacerProps, IGroupSpacerStyleProps, IGroupSpacerStyles>(
-  (props: IGroupSpacerProps) => {
-    const { count, styles, theme, indentWidth = SPACER_WIDTH } = props;
-    const classNames = getClassNames(styles, {
-      theme: theme!,
-      width: count * indentWidth
-    });
+export const GroupSpacer: React.SFC<IGroupSpacerProps> = (props: IGroupSpacerProps): JSX.Element | null => {
```

Comment:
TS should be able to infer typing on the functional args (except for the return type which is currently a [TS bug](https://github.com/Microsoft/TypeScript/issues/241).) Minor point, but could this also be:

```tsx
export const GroupSpacer: React.SFC<IGroupSpacerProps> = (props): ReturnType<React.SFC<IGroupSpacerProps> => {
```

With the `ReturnType` only being required because of the TS bug I mentioned previously. Eventually we should be able to remove that.

Using `ReturnType` is important in case the return type of `React.SFC` changes. If we duplicate its typing, it would result in a build break for us, while `ReturnType` will automatically update.

Reply:
I should clarify, specifying the return type does not change the type of `GroupSpacer`; it is a `React.SFC` either way. However, specifying the return type does ensure the implementation returns the correct type.

Reply:
TIL. Thanks @JasonGore! Done in https://github.com/OfficeDev/office-ui-fabric-react/pull/8435/commits/60d640e344b08b04efabe1e246678970a05698dc.